main.py

```diff
@@ -148,6 +181,15 @@ def setSaveFileName(self, obj):
 		if fileName:
 			obj.setText(fileName)
 
+	def setDirectory(self, obj):
+		options = QFileDialog.Options()
+		directory = QFileDialog.getExistingDirectory(self,
+				"Select ISO Directory",
```

Comment:
Same issue as mentioned before: We not only support ISOs (in fact, the original MS bios does not support ISO[9660] by default as far as I know).